tests/integ/test_multithreading.py

```diff
@@ -32,6 +35,21 @@
 from tests.utils import IS_IN_STORED_PROC, IS_LINUX, IS_WINDOWS, TestFiles, Utils
 
 
+@pytest.fixture(scope="module")
```

Comment:
let's use a function scope here to avoid messing with other tests

Reply:
renamed to threadsafe_session. this would not mess with others, right?